aw_datastore/storages/memory.py

```diff
@@ -13,7 +13,7 @@ class MemoryStorage(AbstractStorage):
     """For storage of data in-memory, useful primarily in testing"""
     sid = "memory"
 
-    def __init__(self, testing):
+    def __init__(self, testing: bool) -> None:
```

Comment:
Isn't that implicit? init can never return something I believe

Reply:
mypy won't typecheck if the function doesn't have a signature, and it requires __init__ to return None.

Reply:
Alright